bundles/org.openhab.binding.awattar/src/main/java/org/openhab/binding/awattar/internal/handler/AwattarPriceHandler.java

```diff
@@ -90,6 +89,7 @@ public void initialize() {
                  * The scheduler is required to run exactly at minute borders, hence we can't use scheduleWithFixedDelay
                  * here
                  */
+                int thingRefreshInterval = 60;
```

Comment:
Maybe better as a static final property of the class?

Reply:
done